cli/dstack/core/job.py

```diff
@@ -352,7 +349,8 @@ def unserialize(job_data: dict):
         app_specs = (
             [
                 AppSpec(
-                    port_index=a["port_index"],
+                    port=a["port"],
```

Comment:
Will it with nicely with the old runs (serialized differently?)

Reply:
I can make it safer, but job resubmitting with ports would not be successful